src/index.ts

```diff
@@ -243,6 +243,6 @@ function Steppp(element: HTMLElement, options: any = defaultOptions) {
   };
 }
 
-Steppp.stepIsValid = (_slide: HTMLElement): boolean => true;
+Steppp.stepIsValid = async (_step: HTMLElement): Promise<boolean> => true;
```

Comment:
Should there be tests for this change?

When I see "Fix" and no tests… it makes me concerned.

Reply:
Ya know what... I dug into this, found out this method is not being used _anywhere_, and it's not mentioned in the public API anywhere. I'm gonna remove it and touch up some tests.